### PR TITLE
FORMS-15150 Runtime client libraries are loaded in authoring for tabs…

### PR DESCRIPTION
**Pull request by dmaurya929**
… and accordion

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/clientlibs/commons/js/common.js [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+                    panel.classList.add(this.constructor.cssClasses.panel.hidden);
+                    panel.classList.remove(this.constructor.cssClasses.panel.expanded);
+                    panel.setAttribute("aria-hidden", true);
+                }
+            }
+        }
+    }
+
+    window.CQ = window.CQ || {};
+    window.CQ.FormsCoreComponents = window.CQ.FormsCoreComponents || {};
```
Comment (rismehta) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/clientlibs/commons/js/common.js:257]: `window.CQ.FormsCoreComponents` we use this namespace only for authoring. We have not had use-cases of commons yet, so let's namespace this separately mentioning the commons.

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/clientlibs/commons/js/common.js [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+                item: {
+                    expanded: "data-cmp-expanded"
+                }
+            };
+
+            constructor(params) {
+                super(params);
+            }
+
+            _getCachedItems() {
```
Comment (rismehta) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/clientlibs/commons/js/common.js:65]: Please use the new way to mention private methods using #

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/clientlibs/editorframe/.content.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    allowProxy="{Boolean}true"
+    categories="[core.forms.components.accordion.v1.editorframe]"
```
Comment (rismehta) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/clientlibs/editorframe/.content.xml:5]: We should add this as part of editor hook. This way we don't need to include it in the component. Client libraries should always be part of header/footer for better performance

**Review by rismehta — CHANGES_REQUESTED** [review]
please add test cases for this change

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/clientlibs/commons/js/common.js (hunk 2 above)
Comment (dmaurya929) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/accordion/v1/accordion/clientlibs/commons/js/common.js:65]: This is base class for `Accordion`, if using `#` then child class would not be able to access. So changed to `_`  just to represent it as private.

**Review by rismehta — CHANGES_REQUESTED** [review]
I had done some changes in tabs, can you incorporate that in this PR, else that use-case would break. Also, could you fix the wizard / accordion issue too.

**Comment by dmaurya929** [comment]
> I had done some changes in tabs, can you incorporate that in this PR, else that use-case would break. Also, could you fix the wizard / accordion issue too.

Updated the PR, also issue is not reproducible in wizard / accordion so doesn't require any changes there.
cc: @rismehta

**Comment by rismehta** [comment]
Please re-base the PR by resolving the conflicts

**Pull request closed without merging**